From 964a6fdee3612f99e12b63b0162df196b2a07fa7 Mon Sep 17 00:00:00 2001
From: Arzhan Kinzhalin <arzhan.i.kinzhalin@intel.com>
Date: Thu, 18 Apr 2019 18:32:48 -0700
Subject: [PATCH v2 2/5 clr-one-shot-updates] clr-one-shot-updates: Autospec creation
 for update from version 2 to version 3

This version deals with the case when upgrade happens when
NetworkManager is not yet installed on the system (configuration is
still created).

Signed-off-by: Arzhan Kinzhalin <arzhan.i.kinzhalin@intel.com>
---
 Makefile                  |  2 +-
 clr-one-shot-updates.spec | 15 ++++++++-------
 options.conf              |  2 +-
 release                   |  2 +-
 upstream                  |  2 +-
 5 files changed, 12 insertions(+), 11 deletions(-)

diff --git a/Makefile b/Makefile
index faf1b15..9c5eb8d 100644
--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
 PKG_NAME := clr-one-shot-updates
-URL = $(CGIT_BASE_URL)/projects/clr-one-shot-updates/snapshot/clr-one-shot-updates-2.tar.bz2
+URL = $(CGIT_BASE_URL)/projects/clr-one-shot-updates/snapshot/clr-one-shot-updates-3.tar.bz2
 ARCHIVES = 
 
 include ../common/Makefile.common
diff --git a/clr-one-shot-updates.spec b/clr-one-shot-updates.spec
index f313bdd..cd243e8 100644
--- a/clr-one-shot-updates.spec
+++ b/clr-one-shot-updates.spec
@@ -3,10 +3,10 @@
 # Generated by: autospec.py
 #
 Name     : clr-one-shot-updates
-Version  : 2
-Release  : 2
-URL      : http://localhost/cgit/projects/clr-one-shot-updates/snapshot/clr-one-shot-updates-2.tar.bz2
-Source0  : http://localhost/cgit/projects/clr-one-shot-updates/snapshot/clr-one-shot-updates-2.tar.bz2
+Version  : 3
+Release  : 3
+URL      : http://localhost/cgit/projects/clr-one-shot-updates/snapshot/clr-one-shot-updates-3.tar.bz2
+Source0  : http://localhost/cgit/projects/clr-one-shot-updates/snapshot/clr-one-shot-updates-3.tar.bz2
 Summary  : No detailed summary available
 Group    : Development/Tools
 License  : GPL-2.0
@@ -24,19 +24,20 @@ license components for the clr-one-shot-updates package.
 
 
 %prep
-%setup -q -n clr-one-shot-updates-2
+%setup -q -n clr-one-shot-updates-3
 
 %build
 export http_proxy=http://127.0.0.1:9/
 export https_proxy=http://127.0.0.1:9/
 export no_proxy=localhost,127.0.0.1,0.0.0.0
 export LANG=C
-export SOURCE_DATE_EPOCH=1553963819
+export SOURCE_DATE_EPOCH=1555634867
+export LDFLAGS="${LDFLAGS} -fno-lto"
 make  %{?_smp_mflags}
 
 
 %install
-export SOURCE_DATE_EPOCH=1553963819
+export SOURCE_DATE_EPOCH=1555634867
 rm -rf %{buildroot}
 mkdir -p %{buildroot}/usr/share/package-licenses/clr-one-shot-updates
 cp COPYING %{buildroot}/usr/share/package-licenses/clr-one-shot-updates/COPYING
diff --git a/options.conf b/options.conf
index b91f539..adbc4f3 100644
--- a/options.conf
+++ b/options.conf
@@ -1,6 +1,6 @@
 [package]
 name = clr-one-shot-updates
-url = http://localhost/cgit/projects/clr-one-shot-updates/snapshot/clr-one-shot-updates-2.tar.bz2
+url = http://localhost/cgit/projects/clr-one-shot-updates/snapshot/clr-one-shot-updates-3.tar.bz2
 archives = 
 giturl = 
 
diff --git a/release b/release
index 0cfbf08..00750ed 100644
--- a/release
+++ b/release
@@ -1 +1 @@
-2
+3
diff --git a/upstream b/upstream
index 0bdd135..ac35d54 100644
--- a/upstream
+++ b/upstream
@@ -1 +1 @@
-dd6195c1cfbe5f594a7b1aaf846dcde8359cbede/clr-one-shot-updates-2.tar.bz2
+83831f369ece28d1ee653bab15dd134c4f18e18e/clr-one-shot-updates-3.tar.bz2
-- 
2.21.0

